test/test-servers/koa_scaffold_server.ts

```diff
@@ -16,11 +16,12 @@
 // jscs doesn't understand koa..
 // jscs:disable
 
-var errorHandler = require('../../src/index.js')({
+import {ErrorReporting} from '../../src/index';
```

Comment:
technically don't need the `/index` at the end :)

Reply:
Done